packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../common/common';
+
+$defaultColor: $ms-color-neutralTertiary;
+$selectedColor: $ms-color-black;
+$hoverColor: $ms-color-themeDarkAlt;
+$disabledColor: $ms-color-neutralTertiaryAlt;
+
+// Hide input field
+.ms-Rating-input {
```

Comment:
Using hidden radio buttons is an interesting approach. I guess the same choices could be presented in a standard radio button list. But wouldn't a `button` for each star be simpler?